client/src/components/DataFiles/DataFilesModals/DataFilesUploadModal.js

```diff
@@ -86,12 +89,13 @@ const DataFilesUploadModal = ({ className, layout }) => {
     setUploadedFiles(files => [...files, ...newFiles]);
   };
 
-  const onRejectedFiles = rejectedFiles => {
+  const onRejectedFiles = newRejectedFiles => {
     const newFiles = [];
-    rejectedFiles.forEach(file => {
+    newRejectedFiles.forEach(file => {
       newFiles.push({ data: file, id: uuidv4() });
     });
     setUploadedFiles(files => [...files, ...newFiles]);
+    setUpRejectedFiles(files => [...files, ...newFiles]);
```

Comment:
Looks like there is an error here which is why you are having to filter out rejected files before uploading:
```  const uploadStart = () => {
    const filteredFiles = uploadedFiles.filter(
      f => status[f.id] !== 'SUCCESS' && !rejectedFiles.includes(f)
    );
```

```suggestion
    setUpRejectedFiles(files => [...files, ...newFiles]);
```
The use of setUploadedFiles here (which i'm suggesting to delete) was a holdover from when the modal still added the too-large files to the list of files to be uploaded.  No reason now to add them to the list of things to be uploaded and then filter them out.